cmd/editorconfig/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bytes"
+	"path/filepath"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/ini.v1"
+
+	ec "github.com/editorconfig/editorconfig-core-go"
+)
+
+const (
+	Version = "1.2.0"
+)
+
+func main() {
+	var (
+		configname string
+		configversion string
+		showVersionFlag bool
+	)
+	flag.StringVar(&configname, "f", ec.ConfigNameDefault, "Specify conf filename other than `.editorconfig'")
+	flag.StringVar(&configversion, "b", "", "Specify version (used by devs to test compatibility)")
+	flag.BoolVar(&showVersionFlag, "v", false, "Display version information")
+	flag.BoolVar(&showVersionFlag, "version", false, "Display version information")
+	flag.Parse()
+
+	if showVersionFlag {
+		fmt.Printf("EditorConfig Core Go, Version %s\n", Version)
+		os.Exit(0)
+	}
+
+	rest := flag.Args()
```

Comment:
Maybe fail if this slice is empty (e.g. user informed no parameter)?

Reply:
Print help and exit when no args are given
https://github.com/editorconfig/editorconfig-core-go/pull/4/commits/42cdfefb7b30639a6997faccf2226b3aec2df37c